src/github3/git.py

```diff
@@ -382,6 +346,42 @@ def recurse(self):
         return self._instance_or_null(Tree, json)
 
 
+class CommitTree(models.GitHubCore):
```

Comment:
Why are you moving this definition? That seems unnecessary

Reply:
This is because of a bug in mypy, let me find the reference.